packages/extension-utils/src/OAuthConfigForm/OAuthConfigForm.tsx

```diff
@@ -119,54 +96,32 @@ export const OAuthConfigForm = ({
   }
 
   const config = getConfig()
-  const [fields, setFields] = useState<IFieldValues>(defaultFieldValues)
-  const [saved, setSaved] = useState<ConfigValues>(config)
-
-  const updateFields = (
-    nameOrValues: string | Partial<IFieldValues>,
-    value = ''
-  ) => {
-    if (typeof nameOrValues === 'string') {
-      setFields((previousFields) => {
-        return { ...previousFields, ...{ [nameOrValues]: value } }
-      })
-    } else {
-      setFields((previousFields) => {
-        return { ...previousFields, ...nameOrValues }
-      })
-    }
-  }
-
-  useEffect(() => {
-    const data = getConfig()
-    const { base_url, looker_url } = data
-    setSaved(data)
-    updateFields({
-      [BASE_URL]: base_url,
-      [WEB_URL]: looker_url,
-      [FETCH_INTENT]:
-        base_url !== '' && looker_url !== '' ? POSITIVE : CRITICAL,
-    })
-  }, [])
-
+  const [apiServerUrlValue, setApiServerUrlValue] = useState('')
```

Comment:
could these be `apiServerUrl` and `setApiServerUrl` instead?

Reply:
I also think the `Value`suffix is redundant